jenkins/init.sls

```diff
@@ -1,5 +1,12 @@
 {% from "jenkins/map.jinja" import jenkins with context %}
 
+make_sure_global_dependencies_are_installed:
+  pkg.installed:
+    - pkgs:
+      - curl
+      - git
+      - java-1.8.0-openjdk
```

Comment:
I added all dependencies for this formula here to make sure they are installed before used.